lib/zendesk_apps_support/manifest.rb

```diff
@@ -0,0 +1,106 @@
+# frozen_string_literal: true
+module ZendeskAppsSupport
+  class Manifest
+    class << self
+      def private_attr_reader(*names)
```

Comment:
No, I was using it in an earlier iteration. Was on the fence whether to keep for future use or not.